slash/site.py

```diff
@@ -24,7 +24,11 @@ def _load_defaults(working_directory=None):
     _load_entry_points()
 
 def _load_slashrc():
-    _load_file_if_exists(os.path.expanduser(config.root.run.user_customization_file_path))
+    user_customization_env_var = os.environ.get("SLASH_USER_SETTINGS")
```

Comment:
@maormarcus a better way to do this is just `os.environ.get('SLASH_USER_SETTINGS', os.path.expanduser(...))`